tests/cypress/integration/stateWorkflow/questionsAnswers/questionAnswers.spec.ts

```diff
@@ -0,0 +1,146 @@
+describe('Q&A', () => {
+    beforeEach(() => {
+        cy.stubFeatureFlags()
+    })
+
+    it('cannot navigate to and from Q&A page with cms-questions flag off', () => {
+        cy.interceptFeatureFlags({
```

Comment:
(issue: blocking question): It seems like there's a very small part of this test's assertions that we are actually using. Can we find a solution where we don't add a test that makes a whole submission to achieve this? I want to avoid adding new standalone tests that are for something small like this because of cypress flakes and runtime.
```
 //Try to visit Q&A page
    cy.visit(`/summary/${submissionId}/q&a`)

     // Look for 404 text on page
      cy.findByRole('heading', { name: '404 / Page not found', level: 1}).should('exist')
```
I suggest attaching those assertions to an existing test for now since we don't have stubbed request yet. For example can we add this to the bottom of  `can navigate to and from Q&A page` and switch the flag right before? Let me know if you think this is workable. I would to more precise and focused with our cypress stuff since we have had ...issues... in the past.

Reply:
Good suggestion!